### DISCUSSION TITLE
Does anyone else get Quota issues?

### DISCUSSION THREAD

**Discussion by manghat** [danielmiessler/Fabric#277] [Q&A]
Error code: 429 - {'error': {'message': 'You exceeded your current quota; please check your plan and billing details. For more information on this error, read the docs: https://platform.openai.com/docs/guides/error-codes/api-errors.', 'type': 'insufficient_quota', 'param': None, 'code': 'insufficient_quota'}}

This is when I set up Fabric for the first time and tried to run it exactly like the [example commands](https://github.com/danielmiessler/fabric?tab=readme-ov-file#example-commands)

**Comment by inphiltrator** [comment]
yep, same here.

**Comment by wsc3** [comment]
Add money to the OpenAI  API account. API usage is different from ChatGPT so add a few $$ to the API account and it should work.

**Comment by rafaelaussie** [comment]
OpenAI API has changed from postpaid to prepaid, so you need to buy the API credit first. It took around 5 minutes for me before the quota is accessible by fabric.
